tests/unit/components/sl-loading-icon-test.js

```diff
@@ -0,0 +1,19 @@
+import Ember from 'ember';
+import { moduleForComponent, test } from 'ember-qunit';
+
+moduleForComponent( 'sl-loading-icon', 'Unit - component: sl-loading-icon' );
+
+test( 'Default classes are set', function( assert ) {
+    var component  = this.subject(),
```

Comment:
Would you rather see

    var $component;
    
    this.subject();
    $component = this.render();
?